### ISSUE TITLE
Post share sometimes leads to main reddit page

### ISSUE THREAD

**Issue by BassTester** [christianselig/apollo-bugs#66]
**Describe the bug**
Using the share button on some posts shares reddit.com instead of the desired post.

**To Reproduce**
1. Go to reddit.com/r/polska
2. Click on 'Share' on any post eg [post](https://www.reddit.com/r/Polska/comments/bqyhbk/kiedy_wiesz_%C5%BCe_i_tak_ci%C4%99_nikt_nie_we%C5%BAmie_na/)
3. Click on 'Reddit Link'
4. Shared content can be copied or passed to other app, but it will pass 'https://reddit.com' instead of the link to post

**iOS version**: 12.3
**Apollo version**: 1.4.18 PRO Enabled
**iOS device**: iPhone 8
**Jailbroken**: no
**Reproducible:** yes, not random

**Expected behavior**
I expected that it would share the post instead of reddit frontpage

**Video**
https://streamable.com/thfkp

**Comment by hutattedonmyarm** [comment]
Happened to me too. Another example post: https://www.reddit.com/r/de/comments/bui8co/der_mittlere_abstand_zweier_fettnäpfchen_ist_seit/

**Comment by eutampieri** [comment]
Yes, with [this post](https://www.reddit.com/r/italy/comments/c801mv/perché_la_gente_vota_lega_e_non_pd/) too. Also,

**iOS version**: 12.3.1
**Apollo version**: 1.4.25
**iOS device**: iPhone 6
**Jailbroken**: no
**Reproducible**: yes, not random

**Comment by doxxx** [comment]
I’ve just experienced this bug as well. Latest Apollo version as of this comment. Latest iOS on iPhone 7s.
